src/test/java/teammates/ui/webapi/FeedbackSessionClosedRemindersActionTest.java

```diff
@@ -105,7 +105,7 @@ public void testExecute() throws Exception {
         action.execute();
 
         // 5 students and 5 instructors in course1
```

Comment:
The comment should be updated as well.

Reply:
Hi Im not sure how to update the comment

Reply:
Well, right now there seems to be some conflicting information. The comment says that there are 10 people, but the next line says 8?  That means that the code and the comment is conflicting.